tools/java/common/test/com/google/i18n/phonenumbers/BuildMetadataFromXmlTest.java

```diff
@@ -617,13 +616,11 @@ public void testAlternateFormatsOmitsDescPatterns()
     PhoneMetadata metadata = BuildMetadataFromXml.loadCountryMetadata("FR", territoryElement,
         false /* liteBuild */, false /* isShortNumberMetadata */,
         true /* isAlternateFormatsMetadata */);
-    assertEquals("(1)(\\d{3})", metadata.getNumberFormat(0).getPattern());
-    assertEquals("1", metadata.getNumberFormat(0).getLeadingDigitsPattern(0));
-    assertEquals("$1", metadata.getNumberFormat(0).getFormat());
-    assertFalse(metadata.hasFixedLine());
-    assertNull(metadata.getFixedLine());
-    assertFalse(metadata.hasShortCode());
-    assertNull(metadata.getShortCode());
+    assertEquals("(1)(\\d{3})", metadata.numberFormat[0].pattern);
+    assertEquals("1", metadata.numberFormat[0].leadingDigitsPattern[0]);
+    assertEquals("$1", metadata.numberFormat[0].format);
+    assertNull(metadata.fixedLine);
```

Comment:
I'm surprised this ever worked - protos should never return null if I remember correctly.
In this case, it should be the empty PhoneNumberDesc, non?